class-auth.php

```diff
@@ -183,7 +183,7 @@ public function get_token( WP_REST_Request $request ) {
 					'success'    => false,
 					'statusCode' => 401,
 					'code'       => $error_code,
-					'message'    => strip_tags( $user->get_error_message( $error_code ) ),
+					'message'    => strip_tags('Authentication failed: '.$error_code),
```

Comment:
The existing code removes HTML tags already, so I'm not sure what this PR actually changes (other than removing a potentially more helpful error message for the end-user)…?